### PR TITLE
Prepare for 2.13.0 development

### PR DESCRIPTION
**Pull request by JohnNiang**
#### What type of PR is this?

/kind cleanup
/area core

#### What this PR does / why we need it:

This PR bumps version to 2.13.0-SNAPSHOT and corrects links of Halo in README.md.

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ docker run -it -d --name halo -p 8090:8090 -v ~/.halo2:/root/.halo2 halohub/halo
 
 ## 生态
 
-可访问 [官方应用市场](https://halo.run/store/apps) 或 [awesome-halo 仓库](https://github.com/halo-sigs/awesome-halo) 查看已经适用于 Halo 2.x 的主题和插件，以及适用于 Halo
+可访问 [官方应用市场](https://www.halo.run/store/apps) 或 [awesome-halo 仓库](https://github.com/halo-sigs/awesome-halo) 查看已经适用于 Halo 2.x 的主题和插件，以及适用于 Halo
```
Comment (ruibaby) [inline README.md:43]:
```suggestion
可访问 [官方应用市场](https://www.halo.run/store/apps) 或 [awesome-halo 仓库](https://github.com/halo-sigs/awesome-halo) 查看适用于 Halo 2.x 的主题和插件。
```

File: README.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ docker run -it -d --name halo -p 8090:8090 -v ~/.halo2:/root/.halo2 halohub/halo
...
 1.x 的相关仓库。
```
Comment (ruibaby) [inline README.md:44]:
```suggestion
```

**Review by ruibaby — CHANGES_REQUESTED** [review]

**Review by guqing — COMMENTED** [review]
/lgtm

**Review by ruibaby — APPROVED** [review]
/lgtm

**Pull request merged**